packaging/tests/tas-restapi/src/test/java/org/alfresco/rest/workflow/deployments/GetDeploymentsSanityTests.java

```diff
@@ -35,7 +35,7 @@ public void getNonNetworkDeploymentsWithAdmin() throws JsonToModelConversionExce
         restClient.assertStatusCodeIs(HttpStatus.OK);
         deployments.assertThat().entriesListIsNotEmpty();
         deployments.getOneRandomEntry().onModel().assertThat()
-                .fieldsCount().is(3).and()
+                .fieldsCount().is(4).and()
```

Comment:
I would just delete checks like this. They're not useful.

Reply:
Oh - unless the idea here is to say "There are only the fields id, deployedAt and name".  What is the 4th field and can we just check the value of it?

Reply:
Added `category` field to the unit test.